project/CONTRIBUTING.md

```diff
@@ -180,9 +180,9 @@ namespace Foo;
 
 interface BarInterface
 {
-
-  // NEXT_MAJOR: Uncomment this method
   /**
+   * NEXT_MAJOR: Uncomment this method
```

Comment:
> This is a bit overkill IMO.

yes, this is too, much, the guy who will release the next major knows how to handle the `NEXT_MAJOR` tags